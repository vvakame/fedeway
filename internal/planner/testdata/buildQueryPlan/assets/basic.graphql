# schema: supergraphSdl.graphqls
{
    me {
        id
        username
    }
}
