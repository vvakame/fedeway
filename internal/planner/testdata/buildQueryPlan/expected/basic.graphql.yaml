node:
  servicename: accounts
  variableusages: []
  requires: []
  operation: |
    query {
    	me {
    		id
    		username
    	}
    }
